time.c

```diff
@@ -4183,17 +4186,23 @@ time_plus(VALUE time1, VALUE time2)
 
 /*
  *  call-seq:
- *     time - other_time -> float
- *     time - numeric    -> time
+ *    self - subtrahend -> new_time or float
```

Comment:
I would keep the two different call sec lines, because they show how the argument type affects the result type. Definitely avoid using `subtrahend`. I can guess the meaning in relation to `addend`, but this is not a word in common use.

Reply:
Fixed.